src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs

```diff
@@ -95,8 +95,12 @@ public void WriteTo(TextWriter writer, HtmlEncoder encoder)
                 throw new ArgumentNullException(nameof(encoder));
             }
 
-            var formattableString = new HtmlFormattableString(Value, _arguments);
-            formattableString.WriteTo(writer, encoder);
+            IHtmlContent htmlContent;
+            if (_arguments != null)
```

Comment:
Braces around if condition blocks:

```C#
if (_arguments != null)
{
   htmlContent = new HtmlFormattableString(Value, _arguments);
}
else
{

}
```

that said, `new HtmlString(value)` assumes `value` is already encoded. Calling `WriteTo` on it writes the value as is. Maybe we don't need that indirection. @dougbu maybe you have some thoughts here.